### DISCUSSION TITLE
Free plan

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#179] [Anything Else]
"Please add a free plan for young sites."

([imported from the old roadmap](https://plausible.nolt.io/151))

**Comment by Lucent** [comment]
Or a free option for open source or non-profit/educational sites.

**Reply by choldgraf** [reply]
We've discussed using plausible for a large open source project I'm involved with, and the price is a non-starter. We have around a million hits a month and as an open source project, have very little funds for something like this :-\

**Reply by metmarkosaric** [reply]
thanks for the feedback! there are no current plans for a free tier unfortunately but for projects that don't mind self-hosting, our self-hosted version is completely free as in beer. see https://plausible.io/self-hosted-web-analytics

**Reply by F-DK** [reply]
@metmarkosaric Any update on this, say for below 10k monthly pageviews?

**Comment by Zwyx** [comment]
Just dropping a note to add my support for a free tier (which would include a few thousand events per months, for instance). Would be very useful for sites not making any money.
